src/buffer/out/textBuffer.cpp

```diff
@@ -201,9 +196,7 @@ ROW& TextBuffer::GetRowByOffset(const til::CoordType index)
 // Returns a row filled with whitespace and the current attributes, for you to freely use.
 ROW& TextBuffer::GetScratchpadRow()
 {
-    auto& row = _getRowByOffsetDirect(0);
-    row.Reset(_currentAttributes);
-    return row;
+    return _getRowByOffsetDirect(0);

```

Comment:
Ah sorry, I forgot to update the comment. I made the change because I wasn't sure what I'd need. Over the weekend I figured that out though, so in the PR I just opened I've fixed this. #15541